OsmAnd-shared/src/commonMain/kotlin/net/osmand/shared/obd/OBDCommand.kt

```diff
@@ -39,5 +39,13 @@ enum class OBDCommand(
 			}
 			return null
 		}
+
+		fun getCommand(name: String): OBDCommand? {
```

Comment:
Better use something like [this](https://github.com/osmandapp/Osmand/blob/dfac98bc21d4bf713c9fe1f7ca298f3c09ac4151/OsmAnd/src/net/osmand/data/SpecialPointType.java#L65)